### PR TITLE
Update in-app_messaging.md

### PR DESCRIPTION
**Pull request by bkasman95**

### CODE REVIEW

File: _docs/_developer_guide/platform_integration_guides/tvos/in-app_messaging.md [hunk 1]
```diff
@@ -22,8 +22,4 @@ Do not choose tvOS from the checkbox list; doing so will prohibit you from lever
 
 ## Customization
 
-Reference our [In-app message custom UI](https://braze-inc.github.io/braze-swift-sdk/documentation/braze/in-app-message-customization) and [Content Cards custom UI](https://braze-inc.github.io/braze-swift-sdk/documentation/braze/content-cards-customization) articles to further customize these channels on tvOS when integrating. We also offer [example projects](https://github.com/braze-inc/braze-swift-sdk/tree/main/Examples) to reference as well to aid in the integration. 
-
-[1]: {% image_buster /assets/img/tvos.png %} 
-[2]: {% image_buster /assets/img/tvos1.png %} 
-[swift-sdk]: https://github.com/braze-inc/braze-swift-sdk
+Braze provided user interface libraries are not available for tvOS and would have to created and managed outside of Braze.
```
Comment (davidbielik) [inline _docs/_developer_guide/platform_integration_guides/tvos/in-app_messaging.md:25]:
```suggestion
Braze provided user interface libraries are not available for tvOS and have to be created and managed outside of Braze.
```

curious why do we not want to point to the custom UI docs like before?

Comment (davidbielik) [reply _docs/_developer_guide/platform_integration_guides/tvos/in-app_messaging.md:25]:
```suggestion
Braze provided user interface libraries are not available for tvOS and would have to created and managed outside of Braze.

Reference our [In-app message custom UI](https://braze-inc.github.io/braze-swift-sdk/documentation/braze/in-app-message-customization) and [Content Cards custom UI](https://braze-inc.github.io/braze-swift-sdk/documentation/braze/content-cards-customization) articles to further customize these channels on tvOS when integrating. We also offer [example projects](https://github.com/braze-inc/braze-swift-sdk/tree/main/Examples) to reference as well to aid in the integration.
```

File: _docs/_developer_guide/platform_integration_guides/tvos/in-app_messaging.md [hunk 2]
```diff
@@ -27,3 +27,5 @@ Reference our [In-app message custom UI](https://braze-inc.github.io/braze-swift
 [1]: {% image_buster /assets/img/tvos.png %} 
 [2]: {% image_buster /assets/img/tvos1.png %} 
 [swift-sdk]: https://github.com/braze-inc/braze-swift-sdk
+
+Braze provided user interface libraries are not available for tvOS and have to be created and managed outside of Braze.
```
Comment (Bucimis) [inline _docs/_developer_guide/platform_integration_guides/tvos/in-app_messaging.md:31]: I'd avoid using passive voice and phrase something like 

"Braze does not provide default UIs for Content Cards or In-App Messages on tvOS. <stuff about customization...>"

**Review by jerielng — APPROVED** [review]

**Review by hokstuff — APPROVED** [review]

File: _docs/_developer_guide/platform_integration_guides/tvos/in-app_messaging.md [hunk 3]
```diff
@@ -22,6 +22,8 @@ Do not choose tvOS from the checkbox list; doing so will prohibit you from lever
 
 ## Customization
 
+Braze does not provide default UIs for Content Cards or In-App Messages on tvOS.
```
Comment (KellieHawks) [inline _docs/_developer_guide/platform_integration_guides/tvos/in-app_messaging.md:25]:
```suggestion
Braze does not provide default UIs for Content Cards or in-app messages on tvOS.
```

**Review by KellieHawks — APPROVED** [review]

**Pull request merged**